cobalt/web/agent.cc

```diff
@@ -628,5 +659,18 @@ void Agent::RequestJavaScriptHeapStatistics(
   callback.Run(heap_statistics);
 }
 
+// Ping watchdog every 5 second, otherwise a violation will be triggered.
+void Agent::PingWatchdog(watchdog::Watchdog* watchdog) {
```

Comment:
This might help responsiveness if we don't want to wait 5s for the very last delayed task to complete. Simply use a non-delayed task with something like a responsive sleep ie. "starboard/common/condition_variable.h" WaitTimed().

https://lbshell-internal-review.git.corp.google.com/c/cobalt_src/+/236082

Reply:
This isn't blocking thread shutdown, nothing waits (nothing should wait) for delayed tasks when a thread is stopped. Delayed tasks to a time after thread shutdown will simply never run.